Wyvarn/calamus
# Token refresh

## Description

Issues new tokens when access token expires.

Fixes #9

## How can we test this?

Ensure application is running with `docker-compose up` & `yarn start` having already been run
Make a request to create a user on endpoint `/api/v1/register`, and then use the access token and refresh tokens from the response to get new tokens as below:

```bash
curl --request POST \
  --url http://localhost:3000/api/v1/token/refresh \
  --header 'authorization: Bearer <ACCESS_TOKEN>' \
  --header 'content-type: application/json' \
  --data '{
	"refreshToken": "<REFRESH_TOKEN>"
}'
```

## Checklist:
